packages/fxa-customs-server/lib/server.js

```diff
@@ -93,6 +93,8 @@ module.exports = async function createServer(config, log) {
   const api = Hapi.server({
     port: config.listen.port,
     host: config.listen.host,
+    // Matches the original idle timeout of the customs-server
+    routes: { timeout: { socket: 120000 } },
```

Comment:
Hrm. https://hapi.dev/api/?v=19.1.1#-routeoptionstimeoutsocket sez the default is already 120000 (2min).

I'm thinking though that this is the wrong setting, not that I know where the right setting is.

Reply:
> not that I know where the right setting is.

Heh, I'm not 100% sure either, but I think we can try this?